WebRoot/js/zimbraMail/mail/view/prefs/ZmAccountsPage.js

```diff
@@ -3212,6 +3212,7 @@ function() {
     } else {
      this._sendAs.focus();
     }
+	this.getButton(DwtDialog.OK_BUTTON).setText(ZmMsg.save);
```

Comment:
I feel it would be better if we would set the label while dialog is being constructed instead of setting it on popup.